### PR TITLE
View/custom colors

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys, I did the custom color implementation, please check it out.

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/CustomJFrame.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * Implements CustomJFrame class.
+ *
+ * @author Maday Alcalá
+ * @version 1.0
+ */
+public class CustomJFrame extends JFrame {
+    JLabel label = new JLabel("Welcome!", JLabel.CENTER);
+    public CustomJFrame(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/CustomJFrame.java]: put space after ")"

File: src/main/java/com/fundation/logic/view/convert/ConvertAudioPanel.java [hunk 2]
```diff
@@ -9,9 +9,7 @@
  */
 package com.fundation.logic.view.convert;
 
-import com.fundation.logic.view.CustomButton;
-import com.fundation.logic.view.CustomPanel;
-import com.fundation.logic.view.PanelSetter;
+import com.fundation.logic.view.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/convert/ConvertAudioPanel.java]: put all specific library

File: src/main/java/com/fundation/logic/view/convert/ConvertBasicPanel.java [hunk 3]
```diff
@@ -9,12 +9,9 @@
  */
 package com.fundation.logic.view.convert;
 
-import com.fundation.logic.view.CustomButton;
-import com.fundation.logic.view.CustomPanel;
-import com.fundation.logic.view.PanelSetter;
+import com.fundation.logic.view.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/convert/ConvertBasicPanel.java]: put all specific library

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java [hunk 4]
```diff
@@ -9,9 +9,7 @@
  */
 package com.fundation.logic.view.convert;
 
-import com.fundation.logic.view.CustomButton;
-import com.fundation.logic.view.CustomPanel;
-import com.fundation.logic.view.PanelSetter;
+import com.fundation.logic.view.*;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: put all specific library

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java [hunk 5]
```diff
@@ -190,22 +188,24 @@ private void videoCriteriaConvertPanel() {
...
-        checkBoxKeyFrame = new JCheckBox("");
+        checkBoxKeyFrame = new CustomCheckBox("");
         GridBagConstraints gbc_checkBoxKeyFrame = new GridBagConstraints();
         gbc_checkBoxKeyFrame.anchor = GridBagConstraints.WEST;
         gbc_checkBoxKeyFrame.insets = new Insets(0, 0, 5, 5);
         gbc_checkBoxKeyFrame.gridx = 2;
         gbc_checkBoxKeyFrame.gridy = 13;
         add(checkBoxKeyFrame, gbc_checkBoxKeyFrame);
 
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: delete unnecessary line

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java [hunk 6]
```diff
@@ -248,22 +240,24 @@ private void videoCriteriaConvertPanel() {
...
-        checkBoxThumbnail = new JCheckBox("");
+        checkBoxThumbnail = new CustomCheckBox("");
         GridBagConstraints gbc_checkBoxThumbnail = new GridBagConstraints();
         gbc_checkBoxThumbnail.anchor = GridBagConstraints.WEST;
         gbc_checkBoxThumbnail.insets = new Insets(0, 0, 5, 5);
         gbc_checkBoxThumbnail.gridx = 2;
         gbc_checkBoxThumbnail.gridy = 14;
         add(checkBoxThumbnail, gbc_checkBoxThumbnail);
 
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: delete unnecessary line

File: src/main/java/com/fundation/logic/view/CustomJFrame.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/CustomJFrame.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertAudioPanel.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertAudioPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertBasicPanel.java (hunk 3 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertBasicPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java (hunk 4 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java (hunk 5 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java (hunk 6 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 7]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
+        gbc_btnSave.insets = new Insets(0, 0, 10, 0);
+        gbc_btnSave.gridx = 4;
+        gbc_btnSave.gridy = 17;
+        add(btnSave, gbc_btnSave);
     }
 
     /**
-     * This method return the CommonSearch Button
+     * This method return the CommonSearch Button.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:304]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 8]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JButton getSearchButton() {
         return btnSearch;
     }
 
     /**
-     * This method return the fieldToDateCreation
+     * This method return the fieldToDateCreation.
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:310]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 9]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
      */
     public JDateChooser getFieldToDateCreation() {
         return fieldToDateCreation;
     }
 
     /**
-     * This method return the FieldFromDateCreation
+     * This method return the FieldFromDateCreation.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:318]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 10]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JDateChooser getTextFieldFromDateCreation() {
         return textFieldFromDateCreation;
     }
     /**
-     * This method return the fieldDateModificationFrom
+     * This method return the fieldDateModificationFrom.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:324]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 11]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JDateChooser getFieldDateModificationFrom() {
         return fieldDateModificationFrom;
     }
 
     /**
-     * This method return the fieldDateModificationTo
+     * This method return the fieldDateModificationTo.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:331]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 12]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JDateChooser getFieldDateModificationTo() {
         return fieldDateModificationTo;
     }
 
     /**
-     * This method return the Field Date Access From
+     * This method return the Field Date Access From.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:338]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 13]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JDateChooser getFieldDateAccessFrom() {
         return fieldDateAccessFrom;
     }
 
     /**
-     * This method return the fieldDateModificationTo
+     * This method return the fieldDateModificationTo.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:345]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 14]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JDateChooser getFieldDateAccessTo() {
         return fieldDateAccessTo;
     }
 
     /**
-     * This method return the Field SizeFrom
+     * This method return the Field SizeFrom.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:352]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 15]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JTextField getTextFieldSizeFrom() {
         return textFieldSizeFrom;
     }
 
     /**
-     * This method return the Field SizeTo
+     * This method return the Field SizeTo.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:359]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 16]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JTextField getTextFieldSizeTo() {
         return textFieldSizeTo;
     }
 
     /**
-     * This method return the ComboBox Read Only
+     * This method return the ComboBox Read Only.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:366]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 17]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JComboBox getComboBoxReadOnly() {
         return comboBoxReadOnly;
     }
 
     /**
-     * This method return the ComboBox Hidden
+     * This method return the ComboBox Hidden.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:373]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 18]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JComboBox getComboBoxHidden() {
         return comboBoxHidden;
     }
 
     /**
-     * This method return the TextField Owner
+     * This method return the TextField Owner.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:380]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 19]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JTextField getTextFieldOwner() {
         return textFieldOwner;
     }
 
     /**
-     * This method return the ComboBox SizeUnit
+     * This method return the ComboBox SizeUnit.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:387]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 20]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public JComboBox getComboBoxSizeUnit() {
         return comboBoxSizeUnit;
     }
 
     /**
-     * This method return the ComboBox Mimetype
+     * This method return the ComboBox Mimetype.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:394]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 21]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
-     * This method set the Field DateModificationFrom
+     * This method return the BtnSearch button.
+     */
+    public CustomButton getBtnSearch() {
+        return btnSearch;
+    }
+
+    /**
+     * This method return the getBtnSave button.
+     */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:408]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 22]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
+    public CustomButton getBtnSave() {
+        return btnSave;
+    }
+
+    /**
+     * This method return the CheckBox ReadOnly.
+     */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:415]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 23]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
+    public JCheckBox getCheckBoxReadOnly() {
+        return checkBoxReadOnly;
+    }
+
+    /**
+     * This method return the CheckBox Hidden.
+     */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:422]: add @return in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 24]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
+    public JCheckBox getCheckBoxHidden() {
+        return checkBoxHidden;
+    }
+
+    /**
+     * This method set the Field DateModificationFrom.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:429]: add @params in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 25]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public void setFieldDateModificationFrom(JDateChooser fieldDateModificationFrom) {
         this.fieldDateModificationFrom = fieldDateModificationFrom;
     }
 
     /**
-     * This method set theField DateModificationTo
+     * This method set theField DateModificationTo.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:436]: add @param in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 26]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
+     * This method set the Field DateAccessFrom.
      */
     public void setFieldDateAccessFrom(JDateChooser fieldDateAccessFrom) {
         this.fieldDateAccessFrom = fieldDateAccessFrom;
     }
 
     /**
-     * This method set the Field DateAccessTo
+     * This method set the Field DateAccessTo.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:450]: add @param in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 27]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public void setFieldDateAccessTo(JDateChooser fieldDateAccessTo) {
         this.fieldDateAccessTo = fieldDateAccessTo;
     }
 
     /**
-     * This method set the TextField SizeFrom
+     * This method set the TextField SizeFrom.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:457]: add @param in documentation.

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 28]
```diff
@@ -265,142 +288,179 @@ public void keyTyped(KeyEvent e) {
...
     public void setTextFieldSizeFrom(JTextField textFieldSizeFrom) {
         this.textFieldSizeFrom = textFieldSizeFrom;
     }
 
     /**
-     * This method set the TextField SizeTo
+     * This method set the TextField SizeTo.
      */
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java:464]: add @param in documentation.

**Pull request merged**